presto-native-execution/presto_cpp/main/operators/BroadcastFactory.cpp

```diff
@@ -29,6 +33,10 @@ namespace {
 std::string makeUuid() {
   return boost::lexical_cast<std::string>(boost::uuids::random_generator()());
 }
+
+const std::string& generateScanId(const std::string& broadcastFilePath) {
```

Comment:
This is not TableScan, hence, "scan ID" is confusing. Let's discuss how to refactor the code offline.